plugins/kubernetes/src/api/KubernetesBackendClient.ts

```diff
@@ -50,11 +50,13 @@ export class KubernetesBackendClient implements KubernetesApi {
     return await response.json();
   }
 
-  async getObjectsByEntity(
-    requestBody: KubernetesRequestBody,
-  ): Promise<ObjectsByEntityResponse> {
+  async getObjectsByLabelSelector(
```

Comment:
Can we keep `getObjectByEntity` and the existing route for querying by entity? We will want more ways to filter objects by entity in the future so this gets a bit too specific